src/append_only_data.rs

```diff
@@ -881,6 +1014,8 @@ impl Data {
         }
     }
 
+    /// Returns a tuple containing the last entries index, last owners index, and last permissions
```

Comment:
We need to explain why it is a `Result` and in which case an error might be returned.

Reply:
Is it intentional that this always returns `Ok`? Can I fix it now or should I write an issue?

Reply:
Good catch :+1: But yes, let's record it as an issue and leave the API as it is for now.

Reply:
Looks like Adam's PR already addressed this :)